src/components/overlays/pause.js

```diff
@@ -17,9 +17,11 @@ export class Pause extends Screen {
     create() {
         this.addBackgroundItems();
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
+        const isAboveSelectScreen = parentKey.includes("select");
```

Comment:
I wonder if getting the scene (via https://photonstorm.github.io/phaser3-docs/Phaser.Scenes.SceneManager.html#getScene or some other route) and seeing if its an `instanceof Select` is a better way? Dunno :shrug: 😄 this seems fine too if the key always has select in.

Reply:
Yeah it is possible to have that flow but this button itself only existed for like two weeks so it's not gonna be the end of the world plus the ticket says don't show it on a select screen so 🤷 😂

Reply:
yeah I think that's a requirement for select screens

Reply:
`instanceof Select` might be clearer though

Reply:
And yeah nick was saying it would be good to move to instanceof checking but a lot of things will have to be moved over to enable the christmas game so this seems ok for now.